server/src/main/java/org/elasticsearch/action/support/SubscribableListener.java

```diff
@@ -34,6 +34,10 @@
  * An {@link ActionListener} to which other {@link ActionListener} instances can subscribe, such that when this listener is completed it
  * fans-out its result to the subscribed listeners.
  * <p>
+ * If this listener is complete, {@link #addListener} completes the subscribing
```

Comment:
The line lengths are a little uneven :) Could you rearrange things to wrap at the 130 character line limit?

Typically folks write comments out to 130 characters, or consistently shorter. This comment uses 130 already.

Reply:
Oops. Ankita was asking me about the other long lines, and I double checked the [contributor's guide](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md) and it's actually 140 characters 😅